src/main/java/edu/ithaca/dragon/blackjack/BlackJack.java

```diff
@@ -4,23 +4,24 @@
 import java.util.Collections;
 
 public class BlackJack {
-    private final int ID;
+    private final long ID;
     private ArrayList<Player> players;
     private ArrayList<Hand> hands;
     private Deck deck;
 
-    public BlackJack(int IDin, Player playerIn){
+    public BlackJack(long IDin, Player playerIn){
         ID = IDin;
         players = new ArrayList<>();
         players.add(playerIn);
         hands = (ArrayList<Hand>)playerIn.getHands().clone();
         deck = new Deck();
     }
-    public BlackJack(int IDin, ArrayList<Player> playerIn)throws IllegalArgumentException{
+    public BlackJack(long IDin, ArrayList<Player> playerIn)throws IllegalArgumentException{
         ID = IDin;
-        Collections.sort(playerIn);
-        for(int x = 0; x < playerIn.size()-1; x++){
-            if(playerIn.get(x).compareTo(playerIn.get(x+1)) == 0)
```

Comment:
Not clear what's happening here or why... Needs a comment to explain

Reply:
Added a comment. 
It sorts a clone of the incoming players to put all duplicates, if there are any duplicates, next to each other. Then compares all sequential pairs for duplicates. The clone of the original array allows for preservation of the order of the incoming players